### ISSUE TITLE
Feature Request: Single logout during logout

### ISSUE THREAD

**Issue by Dzordzu** [wekan/wekan#2905]
## Feature
* Single logout, when using SSO (ex. keycloak endpoint `http://example.com/auth/realms/example/protocol/openid-connect/logout`)
* New env in docker

## Useful links
* [userHeader.js](https://github.com/wekan/wekan/blob/cd253522a305523e3e36bb73313e8c4db500a717/client/components/users/userHeader.js#L30-L44)

**Comment by danielwilson654** [comment]
Yep, this would be handy. Had a look through the code and I don't think it's implemented under the hood so adding just the env wouldn't help.

**Comment by varac** [comment]
Looking forward to this!

**Comment by gitwittidbit** [comment]
+1
